docs/experience/index.md

```diff
@@ -1,6 +1,22 @@
-title: Sematext Experience Overvire
+title: Sematext Experience Overview
 description: Overview of real user monitoring for websites and single page apps
 
-### Overview
+Our Monitoring and Logs solution does a great job of monitoring your infrastructure (ie. your backend) and the software running on it, but for websites and web apps that's only half the story. The other half is of course your frontend and the interaction between the two. Sematext Experience fills this gap by giving you an overview of your websites' performance and the user satisfaction score by measuring directly from your website visitors' browser.
 
-Placeholder
+We believe that it's critical for websites and web apps to load fast in order to keep the visitors satisfied. Recent research<sup>1</sup> has shown that:
+
+ * 53% of visits are likely to be abandoned if pages take longer than 3 seconds to load
+ * One out of two people expect a page to load in less than 2 seconds
+ * 46% of people say that waiting for pages to load is what they dislike the most when browsing the web on mobile devices
+
+### How it works
+
+When you create an Experience App you will be given step-by-step instructions to add and configure our Javascript library to your website. This library is then delivered to your visitors with the rest of your website. The library is small and it's loaded asynchronously so it's not going to affect the performance that you are trying to improve. When the website is ready the library sends the collected performance metrics to Sematext Cloud where you will be able to see and analyze them. The library keeps sending metrics as the website uses ajax requests or on every route change if it's a single page app.
```

Comment:
Maybe add a screenshot of the _RUM Script Installation_ page where you can see the script.
After that add a screenshot of what it looks like after receiving data.

Reply:
Maybe worth mentioning data is sent securely over HTTPS.